From 63fb18773e60de297612f98d749d1791cfd33913 Mon Sep 17 00:00:00 2001
From: Bartosz Fabianowski <bartfab@google.com>
Date: Thu, 25 Jan 2018 11:35:03 +0100
Subject: [PATCH 2/2] Recorder: Request FOREGROUND_SERVICE permission

This is a new permission required to run foreground services. It was
added for internal accounting only - the permission is automatically
granted and not visible to the user. ManagedProvisioning needs it to
support the implementation of silent provisioning, analogously to
commit bda54aa.

Bug: 3404279
Change-Id: Ia6d726928dc2994b611b98c127e20b7cc595aa0c
---
 app/src/main/AndroidManifest.xml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/app/src/main/AndroidManifest.xml b/app/src/main/AndroidManifest.xml
index 1dd8782..67c97a3 100644
--- a/app/src/main/AndroidManifest.xml
+++ b/app/src/main/AndroidManifest.xml
@@ -19,6 +19,7 @@
 
     <uses-permission android:name="android.permission.CAPTURE_SECURE_VIDEO_OUTPUT" />
     <uses-permission android:name="android.permission.CAPTURE_VIDEO_OUTPUT" />
+    <uses-permission android:name="android.permission.FOREGROUND_SERVICE"/>
     <uses-permission android:name="android.permission.READ_PHONE_STATE" />
     <uses-permission android:name="android.permission.RECORD_AUDIO" />
     <uses-permission android:name="android.permission.SYSTEM_ALERT_WINDOW" />
-- 
2.17.1

